pkg/operator/ceph/object/user/controller.go

```diff
@@ -452,13 +452,9 @@ func (r *ReconcileObjectStoreUser) getRgwPodList(cephObjectStoreUser *cephv1.Cep
 
 // Delete the user
 func (r *ReconcileObjectStoreUser) deleteUser(u *cephv1.CephObjectStoreUser) error {
-	_, rgwerr, err := object.DeleteUser(r.objContext, u.Name)
+	output, err := object.DeleteUser(r.objContext, u.Name)
 	if err != nil {
-		if rgwerr == 2 {
-			logger.Infof("ceph object user %q does not exist in store %q", u.Name, u.Spec.Store)
```

Comment:
If the user doesn't exist, don't we still need to ignore this error?

Reply:
If the user does not exist `DeleteUser` returns nil now.

Reply:
good, nice that the third return code is removed as well